src/room.ts

```diff
@@ -399,7 +406,7 @@ export class Room extends EventEmitter implements Sayable {
 
   public member(queryArg: MemberQueryFilter | string): Contact | null {
     if (typeof queryArg === 'string') {
-      return this.member({remark: queryArg}) || this.member({display: queryArg}) || this.member({nick: queryArg})
+      return this.member({alias: queryArg}) || this.member({roomAlias: queryArg}) || this.member({name: queryArg})
```

Comment:
here `alias` & `roomAlias` is a bit confused.

consider to rename to `contactAlias` & `roomAlias` to declare them more clearly.